[2022-07-22 14:12:10,453] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: load_data.create_dataset_table manual__2022-07-22T11:12:08.389579+00:00 [queued]>
[2022-07-22 14:12:10,455] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: load_data.create_dataset_table manual__2022-07-22T11:12:08.389579+00:00 [queued]>
[2022-07-22 14:12:10,455] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 14:12:10,455] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-07-22 14:12:10,455] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 14:12:10,464] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): create_dataset_table> on 2022-07-22 11:12:08.389579+00:00
[2022-07-22 14:12:10,466] {standard_task_runner.py:52} INFO - Started process 18715 to run task
[2022-07-22 14:12:10,468] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'load_data', 'create_dataset_table', 'manual__2022-07-22T11:12:08.389579+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_data_loader.py', '--cfg-path', '/tmp/tmpufjqm_e4', '--error-file', '/tmp/tmpkrdaoe9i']
[2022-07-22 14:12:10,468] {standard_task_runner.py:80} INFO - Job 25: Subtask create_dataset_table
[2022-07-22 14:12:10,477] {logging_mixin.py:115} WARNING - /home/hp/miniconda3/envs/airflow_env/lib/python3.8/site-packages/airflow/configuration.py:532 DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
[2022-07-22 14:12:10,491] {task_command.py:371} INFO - Running <TaskInstance: load_data.create_dataset_table manual__2022-07-22T11:12:08.389579+00:00 [running]> on host hp-HP-Pavilion-Laptop-15-eg0xxx
[2022-07-22 14:12:10,515] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=tesfaye
AIRFLOW_CTX_DAG_ID=load_data
AIRFLOW_CTX_TASK_ID=create_dataset_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-22T11:12:08.389579+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-22T11:12:08.389579+00:00
[2022-07-22 14:12:10,528] {base.py:68} INFO - Using connection ID 'postgres_connection' for task execution.
[2022-07-22 14:12:10,535] {sql.py:255} INFO - Running statement: CREATE TABLE IF NOT EXISTS import_data
(
    track_id bigint,
    " type" text COLLATE pg_catalog."default",
    " traveled_d" double precision,
    " avg_speed" double precision,
    " lat" double precision,
    " lon" double precision,
    " speed" double precision,
    " lon_acc" double precision,
    " lat_acc" double precision,
    "time" double precision,
    other_data text COLLATE pg_catalog."default"
), parameters: None
[2022-07-22 14:12:10,543] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=load_data, task_id=create_dataset_table, execution_date=20220722T111208, start_date=20220722T111210, end_date=20220722T111210
[2022-07-22 14:12:10,559] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 14:12:10,579] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
